src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -177,88 +173,31 @@ private void testCreateInstructorAction() throws Exception {
-        instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackResultsPage.html");
-
-        ______TS("new instructor can edit feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        InstructorFeedbackEditPage feedbackEditPage =
-                instructorHomePage.clickFeedbackSessionEditLink(demoCourseId, "Second team feedback session");
-
-        feedbackEditPage.clickEditSessionButton();
-
-        FeedbackSessionAttributes feedbackSession =
-                getFeedbackSessionWithRetry(demoCourseId, "Second team feedback session");
-        feedbackEditPage.editFeedbackSession(feedbackSession.getStartTime(),
-                                             feedbackSession.getEndTime(),
-                                             new Text("updated instructions"),
-                                             feedbackSession.getGracePeriod());
-        feedbackEditPage.reloadPage();
-        instructorHomePage.verifyHtmlMainContentWithReloadRetry("/newlyJoinedInstructorFeedbackSessionSuccessEdited.html");
-
-        ______TS("new instructor can click submit button of sample feedbackSession");
-        instructorHomePage.loadInstructorHomeTab();
-        FeedbackSubmitPage fbsp = instructorHomePage.clickFeedbackSessionSubmitLink(demoCourseId,
-                                                                                    "Second team feedback session");
-        fbsp.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSubmissionEditPage.html");
-
-        ______TS("new instructor can send reminder of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionRemindLink(demoCourseId,
-                                                          "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionRemind.html");
-
-        ______TS("new instructor can unpublish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionUnpublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionUnpublished.html");
-
-        ______TS("new instructor can publish feedbackSession of sample course");
-        instructorHomePage.loadInstructorHomeTab();
-        instructorHomePage.clickFeedbackSessionPublishLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackSessionPublished.html");
-
-        feedbacksPage.logout();
-
-        ______TS("action failure : invalid parameter");
+        ______TS("action failure : verify the given email is valid or not");
```

Comment:
This is more informative?
`______TS("action failure : trying to create instructor with an invalid email");`